pyproject.toml

```diff
@@ -5,6 +5,7 @@ description = "Official Cognite Data Fusion project templates"
 authors = ["Greger Wedel <greger.wedel@cognite.com>"]
 license = "Apache-2"
 readme = "README.md"
+packages = [{include = "cdf_project_template", from = "."}]
```

Comment:
Should we introduce the cdf-tk script now? And what would be the right package package name? cdf_project_template, cdf_project_templates, or something else then?
Should we explicitly include just the modules and scripts needed instead of everything?